src/templates/config/project/.terrahub.yml.twig

```diff
@@ -3,6 +3,11 @@ project:
   name: '{{ name }}'
   code: '{{ code }}'
   provider: '{{ provider }}'
+  include:
```

Comment:
add this config into `src/templates/config/project/.terrahub.json.twig` as well

Reply:
`include` and `ignore` were added in `.terrahub.json.twig`